src/components/trade/asset-underlying.tsx

```
@@ -140,6 +127,36 @@ const AssetTypeComponent: FC<AssetTypeProps> = ({ selectedAsset, onAssetChange }
         </DropdownMenu>
       </Dropdown>
+
+      {/* Price Display */}
+      <div className="flex items-center space-x-2">
+        {price !== null && (
+          <div className="flex items-center space-x-2">
+            <motion.span
+              className="text-xl sm:text-2xl font-bold px-1 rounded"
+              aria-live="polite"
+              aria-atomic="true"
+              role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)